README.md

```diff
@@ -1,16 +1,189 @@
-# Title
+# Ruby Method Arguments
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Describe the functionality of arguments
+- Define methods that accept arguments
```

Comment:
singular?

Reply:
@sgharms is `Define a method that accepts arguments` better? 
Or should this be `Define methods that accepts arguments`?